src/server/check-answers/controller.test.js

```diff
@@ -0,0 +1,143 @@
+import { createServer } from '~/src/server/index.js'
+import { statusCodes } from '~/src/server/common/constants/status-codes.js'
+import { pageTitle } from './controller.js'
+import { withCsrfProtection } from '~/src/server/common/test-helpers/csrf.js'
+import { parseDocument } from '~/src/server/common/test-helpers/dom.js'
+import SesessionTestHelper from '../common/test-helpers/session-helper.js'
```

Comment:
`SessionTestHelper`?

Reply:
yes - this is the helper i built a couple of weeks ago to store a session within the test so we can use it